torch/_dynamo/variables/builder.py

```diff
@@ -408,9 +408,9 @@ def __init__(
         tx,
         source: Source,
     ) -> None:
-        assert source is not None, (
-            "Consider SourcelessBuilder for ephemeral objects, usually objects created locally."
-        )
+        assert (
```

Comment:
Lots of line errors in this code, please fix
(`lintrunner init`, `lintrunner -a` or `make lint`)

Reply:
The previous state was unlinted, this is after running lintrunner -a.